tests/end_to_end_tests/managed_packages_tests/fixtures/centos6.repo

```diff
@@ -0,0 +1,40 @@
+[C6.10-base]
```

Comment:
CentOS 6 EOL was a couple years ago, is this still needed?

Reply:
We may not explicitly state  that we support CentOS 6, but since new package does not rely one any system dependency except `glibc`, and minimum supported `glibc` version is `2.12`, then still running tests for CentOS 6 should to be a good marker that the package does indeed work properly on systems with such old `glibc`.